pkg/volume/azure_dd/attacher.go

```diff
@@ -181,7 +181,7 @@ func (a *azureDiskAttacher) WaitForAttach(spec *volume.Spec, devicePath string,
 
 		// did we find it?
 		if newDevicePath != "" {
-			// the curent sequence k8s uses for unformated disk (check-disk, mount, fail, mkfs.extX) hangs on
+			// the current sequence k8s uses for unformated disk (check-disk, mount, fail, mkfs.extX) hangs on
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-86f2cf81f39b855ad72e3fac35b08249. Closing.

/close